modules/understanding-upgrade-channels.adoc

```diff
@@ -7,17 +7,156 @@
+although there may be delays while safe paths away from troubled releases are
+constructed and verified.
+
+= Fast and Stable Channel Usage and Strategies
+
+The fast-{product-version} and stable-{product-version} channels present a choice between receiving
+general availability releases as soon as they are available or allowing Red Hat to
+control the rollout of those updates. If issues are detected during rollout, upgrades
+to that version may be blocked in both the fast-{product-version} and stable-{product-version}
+channels, and a new version may be introduced that will be the new preferred upgrade
+target.
+
+Customers can improve this process by configuring pre-production systems on the
+fast-{product-version} channel, production systems on the stable-{product-version} channel,
+and participating in Red Hat’s connected customer program. This program allows Red
+Hat to observe the impact of updates on your specific hardware and software
+configurations. Future releases may improve or alter the pace at which updates move
+from the fast-{product-version} to the stable-{product-version} channel.
+
+If issues are discovered with a release between patch levels, Red Hat may withdraw
+that suggested upgrade path for affected release versions. A newer release would
+become available in the appropriate channels and be suggested as an upgrade path.
+
+= Disconnected Clusters
+
+Customers which have chosen to not be connected to Red Hat and are curating their
+own {product-title} container image content manually should consult the Red Hat
+errata associated with product releases and note any comments impacting upgrades.
+During upgrade the user interface may caution about switching between these versions
+and it is up to the customer to ensure they have correctly selected the appropriate
+version before bypassing those cautions.
+
+= Switching Between Channels
+
+It is supported for customers to switch from the stable-{product-version} channel to
+the fast-{product-versoin} at any time. Customers can also switch to the
+candidate-{product-version} at any time but should be aware that some releases in the
+candidate-{product-version} channel may be release candidates and therefore not
+supported. A customer can switch from candidate-{product-version} to fast-{product-version}
+if their current release is a general availability release. Customers can always
+switch from fast-{product-version} to stable-{product-version}, although there may
+be a delay of up to a day if their current release was recently promoted to
+fast-{product-stable} while they wait for the release to be promoted to
+stable-{product-version}. If you change to a channel that does not include your
+current release, an alert will fire, no updates will be recommended, and you can
+safely change back to your original channel at any point.
+
+Please refer to candidate-{product-version} section above to undertsand the
```

Comment:
@kalexand-rh how do I link to a section in the same adoc file?

Reply:
nit: "undertsand" -> "understand".

> how do I link to a section in the same adoc file?

[Looks like][1] (not sure how stable that anchor will be) `<<${ANCHOR},${TEXT}>>`, so with a BlockId on the header:

```asciidoc
[[candidate-channel]]
= candidate-{product-version}
```

and then consuming text here like:

```asciidoc
Please refer to <<candidate-channel,the candidate-{product-version} section>> to understand...
```

It's also possible that you can just use `<<candidate-channel>>` and get appropriate text filled in automatically.

Although I see stuff like `[id="understanding-upgrade-channels_{context}"]`, and don't know why we'd be using that instead of `[[...]]` BlockIds (they should [be equivalent][2]).

[1]: http://asciidoc.org/userguide.html#X41
[2]: http://asciidoc.org/userguide.html#_block_identifier

Reply:
done